### ISSUE TITLE
Skin modified problems

### ISSUE THREAD

**Issue by Mojas84** [nek7u/NowPlaying-Lite#1]
Hi i add to skin frosted glass plugin with acrylic effect and skin fix for this effect but now is skin bar and buttons and text more in left. Not look good. 
![obrazek](https://github.com/user-attachments/assets/2130e180-15da-4152-a288-4d4e3978b965)

**Comment by Mojas84** [comment]
Skin is super THX. Please we can help me with fix this??

**Comment by nek7u** [comment]
I don't have a plan to implement the forested glass plugin to the skin.
* It seems that the forested glass plugins does not support the rounded corners on Windows 10.
* The **NowPlaying Lite** skin can be changed background rounded corners radius `0-20`, the forested glass plugins can not be changed corners radius.

> but now is skin bar and buttons and text more in left.  

I will see the code If you could share the `.zip` or `.rmskin` of the whole skin files.

* Are you running on Windows 11?
* There are some kind of frosted glass plugins. Which one are you using?

**Comment by nek7u** [comment]
Could you try this?
1. Open `Skins\NowPlaying Lite\Skin.ini` with text editor.
2. Modify  

**From:**
```ini
[Container2]
Meter=Shape
X=(#$M#)R
```
**To:**
```ini
[Container2]
Meter=Shape
X=(2*#$M#)R
```
3. Save file and Refresh skin.

**Comment by Mojas84** [comment]
> I don't have a plan to implement the forested glass plugin to the skin.
> 
>     * It seems that the forested glass plugins does not support the rounded corners on Windows 10.
> 
>     * The **NowPlaying Lite** skin can be changed background rounded corners radius `0-20`, the forested glass plugins can not be changed corners radius.
> 
> 
> > but now is skin bar and buttons and text more in left.
> 
> I will see the code If you could share the `.zip` or `.rmskin` of the whole skin files.
> 
>     * Are you running on Windows 11?
> 
>     * There are some kind of frosted glass plugins. Which one are you using?

I use win 11 rounded corners support. I use this plugin = https://github.com/KazukiGames82/FrostedGlass

**Comment by Mojas84** [comment]
> Could you try this?
> 
>     1. Open `Skins\NowPlaying Lite\Skin.ini` with text editor.
> 
>     2. Modify
> 
> 
> **From:**
> 
> ```ini
> [Container2]
> Meter=Shape
> X=(#$M#)R
> ```
> 
> **To:**
> 
> ```ini
> [Container2]
> Meter=Shape
> X=(2*#$M#)R
> ```
> 
>     3. Save file and Refresh skin.

This not work.

**Comment by Mojas84** [comment]
This is full skin 
[NowPlaying Lite.zip](https://github.com/user-attachments/files/17534800/NowPlaying.Lite.zip)

**Comment by nek7u** [comment]
You have modified `Config.inc` to use FrostedGlass plugin.  
from `$M=(Trunc(10*#$#))` to `$M=(Trunc(0*#$#))`. This will affect whole layout.  
```ini
$=1.03
;; Border width
$B=(Round(#$#-0.25))
;; Margin - Background
$M=(Trunc(0*#$#))
Hidden=1
```
Original - Hold <kbd>Ctrl+Alt</kbd> and click on the skin to highlight
![highlight-original](https://github.com/user-attachments/assets/3127e46a-eb1d-46b2-bdf4-28d8a2357098)
Your modified
![highlight-modified](https://github.com/user-attachments/assets/d1517165-9c41-4a67-a663-fb3a05f5b81f)


Basically Skin users should not modify the `Config.inc` and if you will do modify the `Config.inc` and get problems, you need to solve the problems by your self. 
Hint: Find all `#$M#` in the Skin.ini and adapt to the layout.

If you only need transparent background, you can change tranceparency by Context menu > Styles > Background transparency > 0-100%

**Comment by Mojas84** [comment]
> You have modified `Config.inc` to use FrostedGlass plugin. from `$M=(Trunc(10*#$#))` to `$M=(Trunc(0*#$#))`. This will affect whole layout.
> 
> ```ini
> $=1.03
> ;; Border width
> $B=(Round(#$#-0.25))
> ;; Margin - Background
> $M=(Trunc(0*#$#))
> Hidden=1
> ```
> 
> Original - Hold Ctrl+Alt and click on the skin to highlight ![highlight-original](https://private-user-images.githubusercontent.com/58021194/380582192-3127e46a-eb1d-46b2-bdf4-28d8a2357098.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzAxMDMwNTUsIm5iZiI6MTczMDEwMjc1NSwicGF0aCI6Ii81ODAyMTE5NC8zODA1ODIxOTItMzEyN2U0NmEtZWIxZC00NmIyLWJkZjQtMjhkOGEyMzU3MDk4LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEwMjglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMDI4VDA4MDU1NVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTcwZjBjMTliOTBlNWQyNzg3NTlhOGJlNzEzMDU1YjRhNWU5NDI5YzY0OTRiYzczYmE2MTY5YTE1Yzk4NjhhNTcmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.9rI6RArMspIeXp6_O-Dk6QP0AzML2Z55El2ZwII4mEs) Your modified ![highlight-modified](https://private-user-images.githubusercontent.com/58021194/380582342-d1517165-9c41-4a67-a663-fb3a05f5b81f.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzAxMDMwNTUsIm5iZiI6MTczMDEwMjc1NSwicGF0aCI6Ii81ODAyMTE5NC8zODA1ODIzNDItZDE1MTcxNjUtOWM0MS00YTY3LWE2NjMtZmIzYTA1ZjViODFmLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEwMjglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMDI4VDA4MDU1NVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTkzNjQ0NmNjYjU1NmI0MDc2MzNjMGE2OTU1YmJiMjExNDYxMGE0ZWIwZTliYzQxNTA0NzAzMDY2YWFkNWNkYTEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.9EJbcS6kMxwvIpnlwzC_LudY994LTwa_-R2gYNlocL8)
> 
> Basically Skin users should not modify the `Config.inc` and if you will do modify the `Config.inc` and get problems, you need to solve the problems by your self. Hint: Find all `#$M#` in the Skin.ini and adapt to the layout.
> 
> If you only need transparent background, you can change tranceparency by Context menu > Styles > Background transparency > 0-100%

Hmm sorry i know. I modified to this... But your skin.ini is very complicated write... I not understand as it fix it.

**Comment by Mojas84** [comment]
I change $M=(Trunc(0*#$#)) to $M=(Trunc(3.5*#$#))..... Looks better = 
![obrazek](https://github.com/user-attachments/assets/35db1597-b9a9-4a2c-ad18-802da82c3211)

**Comment by Mojas84** [comment]
> > You have modified `Config.inc` to use FrostedGlass plugin. from `$M=(Trunc(10*#$#))` to `$M=(Trunc(0*#$#))`. This will affect whole layout.
> > ```ini
> > $=1.03
> > ;; Border width
> > $B=(Round(#$#-0.25))
> > ;; Margin - Background
> > $M=(Trunc(0*#$#))
> > Hidden=1
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > Original - Hold Ctrl+Alt and click on the skin to highlight ![highlight-original](https://private-user-images.githubusercontent.com/58021194/380582192-3127e46a-eb1d-46b2-bdf4-28d8a2357098.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzAxMDMwNTUsIm5iZiI6MTczMDEwMjc1NSwicGF0aCI6Ii81ODAyMTE5NC8zODA1ODIxOTItMzEyN2U0NmEtZWIxZC00NmIyLWJkZjQtMjhkOGEyMzU3MDk4LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEwMjglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMDI4VDA4MDU1NVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTcwZjBjMTliOTBlNWQyNzg3NTlhOGJlNzEzMDU1YjRhNWU5NDI5YzY0OTRiYzczYmE2MTY5YTE1Yzk4NjhhNTcmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.9rI6RArMspIeXp6_O-Dk6QP0AzML2Z55El2ZwII4mEs) Your modified ![highlight-modified](https://private-user-images.githubusercontent.com/58021194/380582342-d1517165-9c41-4a67-a663-fb3a05f5b81f.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzAxMDMwNTUsIm5iZiI6MTczMDEwMjc1NSwicGF0aCI6Ii81ODAyMTE5NC8zODA1ODIzNDItZDE1MTcxNjUtOWM0MS00YTY3LWE2NjMtZmIzYTA1ZjViODFmLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEwMjglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMDI4VDA4MDU1NVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTkzNjQ0NmNjYjU1NmI0MDc2MzNjMGE2OTU1YmJiMjExNDYxMGE0ZWIwZTliYzQxNTA0NzAzMDY2YWFkNWNkYTEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.9EJbcS6kMxwvIpnlwzC_LudY994LTwa_-R2gYNlocL8)
> > Basically Skin users should not modify the `Config.inc` and if you will do modify the `Config.inc` and get problems, you need to solve the problems by your self. Hint: Find all `#$M#` in the Skin.ini and adapt to the layout.
> > If you only need transparent background, you can change tranceparency by Context menu > Styles > Background transparency > 0-100%
> 
> Hmm sorry i know. I modified to this... But your skin.ini is very complicated write... I not understand as it fix it.

It have only transparent on picture must add measure to main ini = 
[FrostedGlass]
Measure = Plugin
Plugin = FrostedGlass
Type = Acrylic
Corner = Round
BlurEnabled = 1
BorderVisible = 1
